en/flight_modes/land.md

```diff
@@ -32,24 +32,113 @@ Parameter | Description
 <span id="COM_RC_STICK_OV"></span>[COM_RC_STICK_OV](../advanced_config/parameter_reference.md#COM_RC_STICK_OV) | The amount of stick movement that causes a transition to [Position mode](../flight_modes/position_mc.md) (if [COM_RC_OVERRIDE](#COM_RC_OVERRIDE) is enabled).
 
 
-## Fixed Wing (FW)
+## Fixed-wing (FW)
 
-The vehicle will turn and land at the location at which the mode was engaged.
-RC stick movement is ignored.
+Fixed-wing landing via mav command is currently not supported.

```

Comment:
Are you saying that if you get [MAV_CMD_NAV_LAND](https://mavlink.io/en/messages/common.html#MAV_CMD_NAV_LAND) as a MAVLink command the command will be ignored - is that right? Does that mean the command will just be rejected?

Ditto for setting the mode to LAND using a mode setting command - this will just be rejected and you'll stay in your current mode?

Reply:
I'm currently working on enabling it properly. Currently it's not rejected but broken.

It then will circle down at the current location after getting a LAND command. The flaring part is the same as for the mission lading.